website/docs/reference/project-configs/analysis-paths.md

```diff
@@ -13,12 +13,32 @@ analysis-paths: [directorypath]
 </File>
 
 ## Definition
-Specify a custom list of directories where [analyses](/docs/build/analyses) are located.
+Specify a custom list of directories where [analyses](/docs/build/analyses) are located. 
 
 ## Default
 Without specifying this config, dbt will not compile any `.sql` files as analyses.
 
-However, the [`dbt init` command](/reference/commands/init) populates this value as `analyses` ([source](https://github.com/dbt-labs/dbt-starter-project/blob/HEAD/dbt_project.yml#L15))
+However, the [`dbt init` command](/reference/commands/init) populates this value as `analyses` ([source](https://github.com/dbt-labs/dbt-starter-project/blob/HEAD/dbt_project.yml#L15)).
+
+import RelativePath from '/snippets/_relative-path.md';
+
+<RelativePath 
+path="analysis-paths"
```

Comment:
Will these not work in the code examples? Is there something we can do to make them generic to include them in the snippet?

Reply:
unfortunately they dont seem to work w props. i spent longer than i should yesterday trying to get it to work and found it just easier to hardcode it. i think it could be generic but i found that it's better to be explicit with examples so users avoid having to guess or interpret.